python_modules/dagster/dagster/_core/definitions/antlr_asset_selection/README.md

```diff
@@ -0,0 +1,80 @@
+# How to set up ANTLR
```

Comment:
Do we want to modify this to describe what ANTLR is used for/how to update this for our use-case?